UPGRADE-5.1.md

```diff
@@ -23,6 +23,8 @@ Form
    is deprecated. The method will be added to the interface in 6.0.
  * Implementing the `FormConfigBuilderInterface` without implementing the `setIsEmptyCallback()` method
    is deprecated. The method will be added to the interface in 6.0.
+ * Not defining a third `callable|null $filter` argument to `ChoiceListFactoryInterface::createListFromChoices` and `ChoiceListFactoryInterface::createListFromLoader` methods is deprecated.
```

Comment:
Added argument `callable|null $filter` to `ChoiceListFactoryInterface::createListFromChoices()` and `createListFromLoader()` - not defining them is deprecated.